theano/gof/tests/test_graph_opt_caching.py

```diff
@@ -6,10 +6,8 @@
 import theano
 import theano.tensor as T
 
-
-
 def test_graph_opt_caching():
-    opt_db_file = theano.config.compiledir+'/optimized_graphs.pkl'
+    opt_db_file = theano.config.compiledir+'/*.pkl'
     os.system('rm %s'%opt_db_file)
```

Comment:
We can't change the compiledir during the execution.

On Fri, Jan 16, 2015 at 10:56 AM, abergeron <notifications@github.com>
wrote:

> In theano/gof/tests/test_graph_opt_caching.py
> <https://github.com/Theano/Theano/pull/2299#discussion-diff-23101020>:
>
> >  def test_graph_opt_caching():
> > -    opt_db_file = theano.config.compiledir+'/optimized_graphs.pkl'
> > +    opt_db_file = theano.config.compiledir+'/*.pkl'
> >      os.system('rm %s'%opt_db_file)
>
> You should write setup() and teardown() methods that change the default
> compiledir (perhaps to a subdir of the current one) and make sure to remove
> it after. This way you can isolate the test from the rest of the run.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/2299/files#r23101020>.
>